src/fieri/app/models/publish_worker.rb

```diff
@@ -25,6 +25,10 @@ def perform(cookbook_json, cookbook_name)
       publish_feedback += "#{cookbook_name} is up for adoption\n"
     end
 
+    unless failure == true
+      publish_feedback += "#{cookbook_name} passed the publish metric"
```

Comment:
🤔 While it's not likely that we would end up here with something other than `''` as the value for feedback, maybe an `=` here instead of an append `+=`?

I'm picturing a "no failure? Okay, THIS IS ALL OF YOUR FEEDBACK, GOD JORB" sort of thing.